Client/Client.Qt/Application.cpp

```diff
@@ -26,12 +26,12 @@ void Application::create()
     _icon = new Style::icon(":/images/logo.png");
     _mainwidget->setBioButtonIcon(_icon);
 
+    ConnectionManager::connect();
+    std::thread(&ConnectionManager::loop).detach();
+
     _mainwidget->addWidget(std::make_unique<Login>());
     _mainwidget->addWidget(std::make_unique<Registration>());
-    _mainwidget->addWidget(std::make_unique<ChatWindow>());
```

Comment:
Get it back

Reply:
We cannot create a chat window for a user who has not logged in to the application

Reply:
We can

Reply:
@StasDash give please an example of situation when we create a chat window before or authorization?

Reply:
Caching

Reply:
The situation didn't clear for me. Why we should allow to create a window before authorization? 
As an example, when user manually log out from system, we have a chance to get a data from cash?

Reply:
> When user manually log out from system, we have a chance to get a data from cache? 
Depends on cache mechanism.
Also when user log out from app(app has been not closed), chat window will not be destroyed and after relogin he will use first form created.

Reply:
The chat window may contain requests related to an authorized user, but the user is not yet authorized. We can't create a chat if the user is not logged in. We already had a problem with the fact that we received channels before the user logs in and such errors will appear.

Reply:
> The chat window may contain requests related to an authorized user, but the user is not yet authorized.

After log out (w/o app close) it will.

> We can't create a chat if the user is not logged in.

We can create empty chat. 

> We already had a problem with the fact that we received channels before the user logs in and such errors will appear.

Will be fixed in https://github.com/f1nal3/Juniorgram/issues/210

Reply:
@StasDash 
> chat window will not be destroyed 
Like for me, this is a bad idea. If user log out, this mean we should actually get out, and didn't keep any information in memory.
> and after relogin he will use first form created.
As you said before, "Depends on cache mechanism". We have a serialization mechanism, so we could use it as an example, storing encrypted data in the database. This is safer + faster. 
BUT. 
If user log out, we shouldn't keep any data in active memory.